lkn-form-multilingual-give.php

```diff
@@ -0,0 +1,326 @@
+<?php
+/**
+ * Plugin Name: Form Multilingual for GiveWP
+ * Plugin URI:  https://www.linknacional.com.br/wordpress/givewp/
+ * Description: Add snipets to make Polylang compatible with GiveWP receipts.
+ * Version:     1.0.0
+ * Author:      Link Nacional
+ * Author URI:  https://www.linknacional.com.br
+ * License:     GNU General Public License v2 or later
+ * License URI: http://www.gnu.org/licenses/gpl-2.0.html
+ * Text Domain: lkn-form-multilingual-give
+ */
+
+require_once __DIR__ . '/plugin-updater/plugin-update-checker.php';
+
+// Exit if accessed directly. ABSPATH is attribute in wp-admin - plugin.php
+if (!defined('ABSPATH')) {
+    exit;
+}
+
+/**
+ * Class Lkn_Give_Form_Multilingual
+ */
+final class Lkn_Give_Form_Multilingual {
+    /**
+     * Instance.
+     *
+     * @since
+     * @access private
+     * @var Lkn_Give_Form_Multilingual
+     */
+    private static $instance;
+
+    /**
+     * Give - Tranlate Form Admin Object.
+     *
+     * @since  1.0.0
+     * @access public
```

Comment:
Qual é o impacto de adicionar `@access public`? Isso é necessário?

Reply:
It's not necessary but it's a documentation standard practice